src/clincoded/static/components/variant_central/interpretation/summary.js

```diff
@@ -126,7 +126,9 @@ var EvaluationSummary = module.exports.EvaluationSummary = createReactClass({
                     }
                 }
                 if (!this.state.evidenceSummary) {
-                    this.refs['evaluation-evidence-summary'].resetValue();
+                    if (this.refs['.evaluation-evidence-summary']) {
```

Comment:
During testing, I'm not sure I encountered a scenario where this added condition was necessary (presumably to prevent a fatal crash). Is it actually needed? Is the added period a typo?

Reply:
I think this branch doesn't encounter the issue, because the `evidenceSummary` will always show. I can't remember why I left this line. I believe it is not needed anymore, but I'll double check.

Steps to reproduce issue on dev:
1. Add new variant
2. From evidence view, click interpretation button on the right side
3. Make sure there are no evaluations, and click "View Summary" (error should appear in console)
4. Click ClinGen logo in top left to return to dashboard
5. Expect to see error show up in big text in the middle of the page. (I also got the "View Summary" button to disappear once, but I can't reproduce it.)

As for the period, it is a typo. I apologize for the carelessness

Reply:
On a version of the app running the dev branch, I was able to see the **Cannot read property 'resetValue' of undefined** error when I did a forced refresh on the summary page (of an interpretation with no evaluated criteria). However, as you noted, now that the `evaluation-evidence-summary` field will always be available, the error should no longer appear (without any changes to the code).

In this instance, I think minimizing the amount of code changed (to what's most necessary or pertinent to the task) is preferable to retrofitting the existing code (to avoid opening a can of worms - if we add an existence check for the one field, why not add them for all).